### DISCUSSION TITLE
doc/dev

### DISCUSSION THREAD

**Discussion by giscus** [PickAID/CrychicDoc#91] [Announcements]
# doc/dev

一个包含 Minecraft 开发文档的网站。

https://docs.mihono.cn/zh/doc/dev

**Comment by M1hono** [comment]
你可以使用
```sh
yarn config get registry
```
来查看自己当前使用的源。
然后通过
```sh
yarn config set registry
```
来设置源。

---
当前仍可用的源有：
1. "npm": "https://registry.npmjs.org/",
2. "yarn": "https://registry.yarnpkg.com/",
3. "tencent": "https://mirrors.cloud.tencent.com/npm/",
4. "cnpm": "https://r.cnpmjs.org/",
5. "taobao": "https://registry.npmmirror.com/",
6. "npmMirror": "https://skimdb.npmjs.com/registry/",
7. "ali": "https://registry.npm.alibaba-inc.com/",
8. "huawei": "https://mirrors.huaweicloud.com/repository/npm/",
9. "163": "https://mirrors.163.com/npm/",
10. "ustc": "https://mirrors.ustc.edu.cn/",
11. "tsinghua": "https://mirrors.tuna.tsinghua.edu.cn/"
> [!WARNING]
> 包含官方源。
